app/views/layouts/application.html.erb

```diff
@@ -48,37 +48,39 @@
         <a class="navbar-brand" href="#">Llamame</a>
         <div class="navbar-collapse collapse navbar-responsive-collapse">
           <ul class="nav navbar-nav">
-            <li><%= link_to "Link1", "/path1"  %></li>
-            <li><%= link_to "Link2", "/path2"  %></li>
-            <li><%= link_to "Link3", "/path3"  %></li>
+            <li><%= link_to "Profile", "/users/#{current_user.id}"  %></li>
+            <% if current_user %>
+            <li><%= link_to "Sign Out", "/users/sign_out"  %></li>
+            <% else %>
+            <li><%= link_to "Sign In", "/users/sign_in"  %></li>
+            <% end %>
           </ul>
         </div>
       </div>
     </div>
 
     <div class="container">
       <div class="row">
-        <div class="col-lg-9">
-          <%= bootstrap_flash %>
-          <%= yield %>
-        </div>
         <div class="col-lg-3">
           <div class="well sidebar-nav">
             <h3>Sidebar</h3>
             <ul class="nav nav-list">
-              <li class="nav-header">Sidebar</li>
-              <li><%= link_to "Link1", "/path1"  %></li>
-              <li><%= link_to "Link2", "/path2"  %></li>
-              <li><%= link_to "Link3", "/path3"  %></li>
+              <li class="nav-header">Rename Here</li>
+              <li><%= link_to "Company", "/companies"  %></li>
+              <li><%= link_to "People", "/people"  %></li>
+              <li><%= link_to "Interactions", "/interactions"  %></li>
             </ul>
+            <%= yield :sidebar if content? %>
```

Comment:
This should be something along the lines of `yield(:sidebar) if content_for?(:sidebar)`